finalPrep/modZeroHero.js

```diff
@@ -1,41 +1,57 @@
 // Challenge - See if you can follow the instructions and complete the exercise in under 30 minutes!
 
 // Declare two variables - heroName AND specialAbility - set to strings
+var heroName= 'John Battman'
+var specialAbility = 'have a car with my face on it thats too long to turn.'
 
 // Declare two variables - greeting AND catchphrase
 //   greeting should be assigned to a string that uses concatenation to include the heroName
 //   catchphrase should be assigned to a string that uses interpolation to include the specialAbility
-
+var greeting = "hello I'm "+ heroName
+var catchphrase = `Put a light in a cloud. I'm john batman and I ${specialAbility}`
 // Declare two variables - power AND energy - set to integers
-
+var power = 5
+var energy = 1
 // Declare two variables - fullPower AND fullEnergy
 //   fullPower should multiply your current power by 500
 //   fullEnergy should add 150 to your current energy
-
+var fullPower = power*500
+var fullEnergy= energy+150
 // Declare two variables - isHuman and identityConcealed - assigned to booleans
-
+var isHuman = true
+var identityConcealed = false
 
 // Declare two variables - archEnemies AND sidekicks
 //   archEnemies should be an array of at least 3 different enemy strings
 //   sidekicks should be an array of at least 3 different sidekick strings
-
+var archEnemies = ['Dr Man Battman', 'Mr. Ice Guy', 'Many More']
+//Many More is a villain too.
+var sidekicks = ['Robert', 'Eight Minute Miles', 'Fishy Chip']
 // Print the first sidekick to your console
-
+console.log(sidekicks[0])
 // Print the last archEnemy to the console
-
+console.log(archEnemies[2])
 // Write some code to add a new archEnemy to the archEnemies array
-
+archEnemies.unshift('Questions Mark')
```

Comment:
Usually, we add new elements to the end of the array using the `push` method. The `unshift` method will add the element to the beginning of the array, which throws off all of the other index values of other elements.